debian/rules

```diff
@@ -1,18 +1,50 @@
 #!/usr/bin/make -f
+#
+# Build Debian package using https://github.com/spotify/dh-virtualenv
+#
+# The below targets create a clean copy of the workdir via
+# using "sdist", else "pip" goes haywire when installing from
+# sourcedir ".", because that includes the debian build stage,
+# and a recursive explosion ensues when symlinks are followed.
+#
+# It also ensures your MANIFEST is complete and at least covers
+# all files needed for a release build.
 
-# This file was automatically generated by stdeb 0.8.2 at
-# Fri, 12 Jun 2015 14:32:03 +0100
-export PYBUILD_NAME=matrix-synapse
-%:
-	dh $@ --with python2 --with systemd --buildsystem=pybuild --no-guessing-deps
+# Increase trace logging, see debhelper(7) (uncomment to enable)
+#DH_VERBOSE=1
+
+export DH_VIRTUALENV_INSTALL_ROOT=/opt/venvs
+export CFLAGS=-march=x86-64
+SNAKE=/usr/bin/python3
+DEB_BUILD_OPTIONS=noddebs
```

Comment:
what does this do?

Reply:
(I think it's a deprecated option for dh_strip, but since we're disabling dh_strip anyway, it seems redundant?)

Reply:
removed